lib/galaxy/config/sample/job_conf.xml.sample_advanced

```diff
@@ -604,6 +604,12 @@
         </destination>
         <destination id="singularity_local" runner="local">
           <param id="singularity_enabled">true</param>
+          <!--Galaxy requests an isolated /tmp directory from singularity, which means
```

Comment:
Is the `read-only` part of this statement true)? 

See discussion in https://github.com/galaxyproject/galaxy/issues/15673  and tests that have been added in https://github.com/galaxyproject/galaxy/pull/15614

Reply:
From the discussion you shared 

> instead (by default) provide a writable /tmp via $_GALAXY_JOB_TMP_DIR/:/tmp:rw note that this dir is still also available as $_GALAXY_JOB_TMP_DIR:rw

I found that this /tmp mount is only provided when `tmp_dir` is set to true. Is that intentional behavior?

Reply:
> I found that this /tmp mount is only provided when tmp_dir is set to true. Is that intentional behavior?

Guess so... and if one of `TMP`, `TEMP`, `TMPDIR` is `/tmp`, or?

Regarding FastQC: we should set `--dir` in the tool wrapper: https://github.com/s-andrews/FastQC/blob/1faeea0412093224d7f6a07f777fad60a5650795/fastqc#L480 .. the culprit here seems to be JAVA (just another reason to dislike it).

Reply:
https://github.com/galaxyproject/tools-iuc/pull/6207